lib/views/general/frontpage.html.erb

```diff
@@ -41,6 +41,11 @@ valueOf()+(h?'&utmxhash='+escape(h.substr(1)):'')+
   </div><%# close div#content %>
   </div><%# close div#wrapper %>
 
+<% end %>
+
+<% cache_if_caching_fragments_conditional(!@blog_items.empty?,
+                                          "latest_blog_posts-#{@locale}",
+                                          :expires_in => 4.hours) do %>
```

Comment:
It's [good practice](http://guides.rubyonrails.org/v3.2.22/caching_with_rails.html#activesupport-cache-store) to use `: race_condition_ttl` when setting `:expires_in`.